eZ/Publish/SPI/Limitation/Target/Builder/VersionBuilder.php

```diff
@@ -124,6 +134,11 @@ function (Field $field) {
             array_unique($languageCodes)
         );
 
+        if (!empty($this->content)) {
+            dump($fields);
+            dump($this->content);
+        }
+
```

Comment:
This is not gonna work in prod :P

Reply:
Oops. Squashed dev commit by accident :P